src/main/java/com/sinthoras/visualprospecting/mixins/xaerosworldmap/xaerosminimap/WaypointsIngameRendererMixin.java

```diff
@@ -0,0 +1,36 @@
+package com.sinthoras.visualprospecting.mixins.xaerosworldmap.xaerosminimap;
+
+import com.sinthoras.visualprospecting.gui.xaeromap.XaeroMapState;
+import com.sinthoras.visualprospecting.gui.xaeromap.waypoints.WaypointManager;
+import net.minecraft.client.Minecraft;
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Slice;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import xaero.common.XaeroMinimapSession;
+import xaero.common.minimap.waypoints.render.WaypointsIngameRenderer;
+
+@Mixin(value = WaypointsIngameRenderer.class, remap = false)
+public class WaypointsIngameRendererMixin {
+
+	@Inject(method = "render",
+			at = @At(value = "INVOKE",
+					target = "Lxaero/common/minimap/waypoints/render/WaypointsIngameRenderer;renderWaypointsList(Ljava/util/Collection;DDDLnet/minecraft/entity/Entity;Lnet/minecraft/client/renderer/Tessellator;DDDDFFDIFFLnet/minecraft/util/Vec3;D)V"
+			),
+			slice = @Slice(
```

Comment:
Thanks for bringing @Slice to my attention! I never knew...

Reply:
Same for @Unique :-)

Reply:
I'd recommend giving [the fabric wiki's mixin section](https://fabricmc.net/wiki/tutorial:mixin_introduction) a look if you haven't already, good amount of useful information there

Reply:
though do keep in mind it's written with the fabric loader and mixin 0.8 in mind